include/dfu/pcd.h

```diff
@@ -54,9 +54,20 @@ extern "C" {
 enum pcd_status {
 	PCD_STATUS_COPY = 0,
 	PCD_STATUS_COPY_DONE = 1,
-	PCD_STATUS_COPY_FAILED = 2
+	PCD_STATUS_COPY_FAILED = 2,
+	PCD_STATUS_READ_VERSION = 3,
+	PCD_STATUS_READ_VERSION_DONE = 1,
+	PCD_STATUS_READ_VERSION_FAILED = 2,
```

Comment:
I have a wage memory that I've read somewhere that this is a bad idea but I can't remember where. I do find it more readable to have duplicate enum values with regards to the code.